lib/pages/chat/events/html_message.dart

```diff
@@ -525,3 +553,15 @@ class MatrixPill extends StatelessWidget {
     );
   }
 }
+
+FontSize? tryGetParentFontSize(ExtensionContext context) {
```

Comment:
This function is basically the relevant part scaling custom emotes into the right parenting text size. I'm sure it could be better or "more complete" but from everything I see as of now how we use font sizes in the HTML renderer, it should work in any occurring case.